_pages/professional-services.md

```diff
@@ -0,0 +1,80 @@
+available.
+
+In addition to the best-effort support available through these (support channels)[/documentation/#community-support/], 
+our MISP Professional Services are meant to be used by companies and organisations who want to make sure their issues are 
+fast-tracked and receive higher importance.
+
+The following MISP Professional Services are available: 
+
+## Trainings
+
+The MISP Project team is very experienced doing specific trainings for small groups and larger organisations and audiences, 
+either at our location or at a specified location of the organisation taking the training.
+We are flexible on the content and can adopt it to the level of expertise of the participants if required.
+Most of the time we are doing sessions of 1 to 3 days, with raising level of technical experience each day.
+The training is always held by two of our MISP core team.
+
+The MISP team gives frequently open trainings within their premises in Luxembourg and sometimes abroad.
+A list of current trainings can be found at the [events page](/events/).
+
+There are different ways of custom trainings:
+
+a) For on-site closed trainings up to 30-50 attendees (depending on conditions of the location): 
+We charge EUR 3.000 per training day. 
+
+b) For closed trainings up to 30 attendees in Luxembourg: 
+We charge EUR 3.500 per training day including food and beverages
+
+
+## Support contract
+
+We offer a yearly support contract for the fast-tracked resolution of 5 issues.
+This contract includes the prioritised resolution of 5 issues, configuration requests or fast-tracked bug fixes for the MISP core software or modules maintained by us. The support is given through our ticket system within business hours.
+Fast-tracking a new/custom feature is usually project based and not included in this support contract, but depending on the complexity it could be handled within this contract. This kind of support costs EUR 6.000 per year.
+
+## Sustainability support
+
+In order to support the sustainability of MISP as an Open Source project, we offer the display of your logo prominently on the MISP Project website's supporter section. This kind of support costs EUR 9.000 per year.
+
+## Support contract with sustainability support
+
+In order to support the sustainability of MISP as an Open Source project, we offer the display of your logo prominently on the MISP Project website's supporter section. In addition, this contract includes the prioritised resolution of 6 issues, configuration requests or fast-tracked bug fixes for the MISP core software or modules maintained by us. The support is given through our ticket system within business hours.
+Fast-tracking a new/custom feature is usually project based and not included in this support contract, but depending on the complexity it could be handled within this contract. This kind of combined support costs EUR 15.000 per year.
```

Comment:
but depending on the complexity of the requested feature, exceptions can of course be made. This type of combined support...